indexer/src/index_core/src101.py

```diff
@@ -543,15 +545,12 @@ def handle_mint(self):
                     recipient_nvalue=self.src101_dict.get("destination_nvalue"),
                 )
                 return
-            # check img
+            # set img
             if self.imglp and self.imgf:
+                if len(self.src101_dict.get("tokenid")) > len(self.src101_dict.get("img")):
```

Comment:
Why don't you just change it to !=

Reply:
When the number of `img` exceeds the number of `tokenid`, we still consider the transaction valid. To enhance understanding, we will adopt a more user-friendly approach to accommodate more scenarios.